### PR TITLE
EXP-13297 - UI Tests: One drive: No sound is playing

### PR DESCRIPTION
**Pull request by FedirKavun**
__Ticket link__

[EXP-13297](https://readdle-j.atlassian.net/browse/EXP-13297)
[EXP-13296](https://readdle-j.atlassian.net/browse/EXP-13296)

__PR description__

- Deleted the skip authentication because the logic in RSOneDriveOnlineClient and RSOffice365SharepointOnlineClient changed

__PR submission checklist__

- [x] PR name contains Jira ticket number
- [x] PR have correct target branch
- [x] .s7substat has correct subrepos revisions
- [x] Common ([Obj-C](https://readdle-c.atlassian.net/wiki/spaces/DOC/pages/3802038524/Common+Objective-C+Code+Style+Guide+for+Docs+PE+iOS+Mac+Teams) / [Swift](https://readdle-c.atlassian.net/wiki/spaces/DOC/pages/4662394906/Common+Swift+Code+Style+Guide+for+Docs+PE+iOS+Mac+Teams)) and your team's coding conventions and style guidelines are followed
- [x] Unit tests to cover the critical parts of the code are written
- [x] Relevant documentation updated / added if needed
- [x] Self-reviewed using the [self-review checklist](https://readdle-c.atlassian.net/wiki/spaces/DOC/pages/4645978135) prior to submitting a PR
- [x] All recommendations from the [How to create Pull Request](https://readdle-c.atlassian.net/wiki/spaces/DOC/pages/4568416271/How+to+create+Pull+Request+PR) document are followed

[EXP-13297]: https://readdle-j.atlassian.net/browse/EXP-13297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EXP-13296]: https://readdle-j.atlassian.net/browse/EXP-13296?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: MSGraphSDK/MSGraphCoreSDK/requests/MSGraphDriveItemContentRequest.h [hunk 1]
```diff
@@ -9,7 +9,7 @@ typedef void(^MSGraphDriveItemUploadCompletionHandler)(MSGraphDriveItem *respons
 
 @interface MSGraphDriveItemContentRequest : MSRequest
 
-- (instancetype)initWithDownloadURL:(NSURL *)url client:(ODataBaseClient *)client;
+- (NSMutableURLRequest *) download;
```
Comment (pastey) [inline MSGraphSDK/MSGraphCoreSDK/requests/MSGraphDriveItemContentRequest.h:12]: nit: предлагаю не вытягивать этот метод наружу. Обойтись в клиентских кодах `requestWithMethod:@"GET" ...`.
В крайнем случае, переименовать его в что-то типа `createDownloadRequest`

**Review by pastey — CHANGES_REQUESTED** [review]
Все нравится. Одно предложение.

File: MSGraphSDK/MSGraphCoreSDK/requests/MSGraphDriveItemContentRequest.h (hunk 1 above)
Comment (FedirKavun) [reply MSGraphSDK/MSGraphCoreSDK/requests/MSGraphDriveItemContentRequest.h:12]: Согласен, уже поменял

**Review by pastey — APPROVED** [review]
Дякую!

**Pull request closed without merging**